flows/webhook_test.go

```diff
@@ -227,30 +227,42 @@ func TestLegacyWebhookPayload(t *testing.T) {
 		"results": {
 			"2factor": {
 				"category": "",
+				"categories": [""],
```

Comment:
I looked at trying to keep the legacy webhook payload the same but we don't have the right functions in excellent to pull it off - `foreach` only works on arrays, not objects like `@results`

So... maybe we just accept a certain amount of duplication here 🤷‍♂. We're only adding fields so this shouldn't break anyone's stuff.

Reply:
I can't believe you didn't add foreach support for dicts! But ya, I'm ok with this.